docs/user-guide/src/version_support.md

```diff
@@ -55,18 +55,16 @@ Following table summarizes BMO release/test process:
 | v0.8          | Supported |
 | v0.6          | Supported |
 | v0.5          | Tested    |
-| v0.4          | Tested    |
+| v0.4          | EOL       |
 | v0.3          | EOL       |
 | v0.2          | EOL       |
 | v0.1          | EOL       |
 
 ## Ironic-image
 
 Since `v23.1.0`, Ironic follows the semantic versioning scheme for its own
-release cycle, the same way as CAPM3 and IPAM. Currently, we have release-26.0,
-release-25.0, release-24.1, release-24.0 and release-23.1 release branches for
-`v26.0.x`, `v25.0.x`, `v24.1.x` `v24.0.x` and `v23.1.x` release cycle respectively
-and as such two or three braches are maintained as supported releases.
+release cycle, the same way as CAPM3 and IPAM. Currently, we have release-26.0, release-25.0, release-24.1, release-24.0 and release-23.1 release branches for
```

Comment:
23.1 still here, and now the line is long too. Remove it and rewrap line to 80.

Reply:
Afaiu, here we just list the branches, not just the ones in support? `release-23.1` branch is still there in the repo, why do we remove it here?

Reply:
We have table underneath which has all the branches. We don't actually need to list any of these. If you take a look at CAPM3 or IPAM, we just give example whats maintained and whats not.

BMO actually has the same issue as Ironic-image and it is listing the branches for no value, and is also being outdated now.